ddpui/ddpairbyte/airbyte_service.py

```diff
@@ -29,13 +33,32 @@
 
 def abreq(endpoint, req=None, **kwargs):
     """Request to the airbyte server"""
+    request = thread.get_current_request()
+
     abhost = os.getenv("AIRBYTE_SERVER_HOST")
     abport = os.getenv("AIRBYTE_SERVER_PORT")
     abver = os.getenv("AIRBYTE_SERVER_APIVER")
     token = os.getenv("AIRBYTE_API_TOKEN")
 
-    logger.info("Making request to Airbyte server: %s", endpoint)
+    if request is not None:
+        org_user = request.orguser
 
+        if flag_enabled("AIRBYTE_PROFILE", request_org_id=str(org_user.org.id)):
+            org_slug = org_user.org.slug
+            block_name = f"{org_slug}-{slugify(AIRBYTESERVER)}"
```

Comment:
We should get the airbyte server block from the `OrgPrefectBlockv1` table and use the `block_name` stored there. Each org will have one.

Reply:
Updated